src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -288,14 +291,19 @@ export const TradeInput = (props: CardProps) => {
             </Stack>
             <Stack px={4}>
               <DonationCheckbox isLoading={isLoading} />
+              {rate && (
```

Comment:
maybe we can use the existence of the quote as the conditional here rather than a nested property?
```suggestion
              {activeQuote && (
```

Reply:
That's a smarter approach! Addressed in https://github.com/shapeshift/web/pull/4906/commits/a69f7bbdfbcfeca1ac22fa907abfc2ad69ab3651.